<!--%Tag(FULL)%-->
<launch>
  
	<param name="bextract_args" value="-mfcc" />
  <param name="feature_subs" value="1" />
  <param name="filter_peaks" value="false"/>
  <param name="input_mode" value="live" />
  <param name="gesture_topic" value="gesture_vector" />
  <param name="music_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/dir/music" />
  <param name="pca_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/dir/pca" />
  <param name="sample_duration" value="2" />
  <param name="sample_log" value="100" />

  <node pkg="audiogesture" type="audio_gesture_server" name="AudioGestureServer" output="screen"/>

  <node pkg="audiogesture" type="music_extractor" name="MusicFeatureExtractor" output="screen"/>

  <node pkg="audiogesture" type="music_extractor_buffer" name="MusicFeatureExtractorBuffer" output="screen"/>

  <node pkg="audiogesture" type="sample_listener" name="SampleListener" output="screen"/>

  <node pkg="audiogesture" type="collection_generator" name="CollectionGenerator" output="screen"/>

  <node pkg="audiogesture" type="feature_normalizer" name="FeatureNormalizer" output="screen"/>
  
  <node pkg="audiogesture" type="gesture_visualizer" name="GestureVisualizer" output="screen"/>
  
  <node pkg="audiogesture" type="audio_input" name="AudioInput" output="screen"/>
  
</launch>
<!--%EndTag(FULL)%-->

<!--
  
  
-->
